man/fi_domain.3.md

```diff
@@ -581,6 +583,17 @@ level features that are supported by the provider.
 
 The operational mode bit related to using the domain.
 
+## default_auth_key
+
+The default authorization key to associate with endpoint and memory 
+registrations created within the domain.
+
+## default_auth_keylen
+
+The length of the default authorization key for the domain. If set to 0, then
+there no authorization key will be associated with endpoints and memory 
+registrations created within the domain.
+
```

Comment:
Similar comment for mr_attr values.  And for consistency, please add a statement that the auth_key fields (domain and endpoint) are ignored unless the API is 1.5 or greater.  The MR definitions already include this.  Thanks